newrelic/structures_newrelic_nrql_alert_condition.go

```diff
@@ -491,7 +698,8 @@ func flattenNrql(nrql alerts.NrqlConditionQuery, configNrql map[string]interface
 
 	// Handle deprecated
 	if svRaw != nil && svRaw.(string) != "" {
-		out["since_value"] = strconv.Itoa(nrql.EvaluationOffset)
+		evalOffset := nrql.EvaluationOffset
+		out["since_value"] = strconv.Itoa(*evalOffset)
```

Comment:
`EvalOffset` is [now a pointer.](https://github.com/newrelic/newrelic-client-go/pull/805/files#diff-a8983aa7bc4d2d7b3c0924422a5a2f907a76fd3eb9a2a3e8c6581cdfeef8d4ddR227)